ARCHITECTURE.md

```diff
@@ -45,27 +48,27 @@ Kedro-Viz loads data asynchronously in production from the API, or when using th
 
 Each pipeline endpoint corresponds to a different top-level pipeline. Only one top-level pipeline should be loaded at a time, so loading data from a pipeline endpoint will reset the pipeline state in the store. Each pipeline dataset contains all the data required to render the graph.
 
-On first page-load, the app always loads the `/api/main` endpoint first. This is the endpoint that corresponds to the 'default' pipeline. The app can load other pipelines from `/api/pipeline/<id>`. If another pipeline is saved as the active pipeline in localStorage, and if it exists in the current project, then the app will load that pipeline on first page-load, but it will always load the `/api/main` endpoint first, in order to check whether the active pipeline is present at that endpoint before requesting it.
+On first page-load, the app always loads the `/api/main` endpoint first. This is the endpoint that corresponds to the 'default' pipeline. The app can load other pipelines from `/api/pipeline/<id>`. If another pipeline is saved as the active pipeline in local storage, and if it exists in the current project, then the app will load that pipeline on first page load, but it will always load the `/api/main` endpoint first, in order to check whether the active pipeline is present at that endpoint before requesting it.
 
 ### Node API endpoints
 
 Each node endpoint contains data required to populate the metadata panel for that node. When a user selects a node on the graph, if data for this node is not already present, then the app will request additional node data from `/api/nodes/<id>`.
 
-## localStorage
+## `localStorage`
```

Comment:
Technically this is more correct, but backticked code snippets in headings look absolutely dreadful with the Github formatting! localStorage is pretty well-known to front-end devs and I think in this case it's sufficiently recognisable without the formatting?